### ISSUE TITLE
Clarification needed on training Probing-RAG with multiple QA pairs and hyperparameter discrepancies

### ISSUE THREAD

**Issue by CHEN010325** [baekingeol/Probing-RAG#4]
<img width="393" height="92" alt="Image" src="https://github.com/user-attachments/assets/865993bb-b5cb-451a-985d-82feddab9920" />
Hello,

I have a couple of questions regarding training the Probing-RAG prober model:

In my experiments, I noticed that the training seems to use only one QA pair per sample, either "no retrieval" or "simple retrieval" cases. Could you please clarify how you train the prober using both retrieval and non-retrieval QA pairs simultaneously in your dataset? I am not fully understanding this aspect.

Regarding the hyperparameters: In the paper, the learning rate is reported as 1e-3 and batch size as 12, but in the released code the learning rate is set to 1e-4 and batch size is 8. Could you please explain why there is this discrepancy?

Thank you very much for your time and support.

Best regards.

**Comment by baekingeol** [comment]
I apologize for the late response.

Please return all of the hotpot, nq, and trivia data and merge them.

Also, the hyperparameters listed in the paper are accurate.

Best regards.
